### DISCUSSION TITLE
Set temperature and hold until next

### DISCUSSION THREAD

**Discussion by sdahbour** [dahlb/ha_carrier#133] [Q&A] [answered]
Currently when you set the temperature in Home Assistant, it ends up setting the temperature and holding forever.  Is there any way to change this so that it changes the temperature and holds until the next scheduled profile?  (This is what the iOS Carrier Home app does, and what the actual thermostat does.)  

So the scenario would be that I have the Sleep profile set to start at 11PM (temp 75 degrees).  It's currently 8PM and the Home profile is running (temp 73 degrees).  I change the temperature to 72 degrees through Home Assistant (through ha_carrier).  At this point, the thermostat is in Manual mode and will maintain 72 degrees indefinitely.  Instead, I'd like the schedule to kick back in at 11PM, when the profile should change to Sleep.  (Again, this is actually how the thermostat and the Carrier Home app both work, so it seems like it should be possible.)  

Or maybe I've missed something and there's already a way to do this?  Thanks!!!

**Comment by dahlb** [comment, marked as answer]
This is an existing feature, ensure you have it enabled in your integration settings

![Screenshot_20240730-115102](https://github.com/user-attachments/assets/f25c8a66-20cd-40ef-8fab-7aa9888478e6)

**Reply by sdahbour** [reply]
THANK YOU!!! I didn’t see that!  Thanks again!

**Reply by sdahbour** [reply]
Unfortunately, that didn’t work.  I unchecked the checkbox.
![IMG_0676](https://github.com/user-attachments/assets/e58217db-165a-4002-86ba-ca244e61915b)

Then I change the temperature from 75 to 74 in Home Assistant.  The label changes to “manual” at this point.
![IMG_0677](https://github.com/user-attachments/assets/e7cf0248-455e-4fed-a21e-8ab81f0d9dc8)

Then the thermostat and Carrier app show that the temperature is holding.  At this point, it is on hold indefinitely.
![IMG_0678](https://github.com/user-attachments/assets/4d925e3d-5c9b-4222-a999-a9749b0c94e6)

Sorry about that.
